examples/radial_histogram_equalization.py

```diff
@@ -0,0 +1,140 @@
+"""
+===================================
+Radial Histogram Equalization
+===================================
+
+This example applies the radial histogram equalizing filter (`sunkit_image.radial.rhef`) filter to a sunpy map.
+"""
+
+import astropy.units as u
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import sunpy.data.sample
+import sunpy.map
+
+import sunkit_image.enhance as enhance
+import sunkit_image.radial as radial
+from sunkit_image.utils import equally_spaced_bins
+
+#######################################################################################
+# Let us us the sample AIA image to test the RHE filter
+
+# Load AIA image
+aia_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+
+# Create radial segments (RHEF should use a dense grid)
+radial_bin_edges = equally_spaced_bins(0, 2, aia_map.data.shape[0] // 2)
+radial_bin_edges *= u.R_sun
+
+# Apply the radial filter to the map
+rhef_map = radial.rhef(aia_map, radial_bin_edges)
+
+
+#######################################################################################
+# It seems that the native sunpy plot routine has a strong effect on the output,
+# so we recommend using imshow.
```

Comment:
What are they? I would rather we use the map plot and make changes to the output that way than use imshow.

Reply:
check out the 3-column plot on this page, which demonstrates the difference.

https://sunpy--205.org.readthedocs.build/projects/sunkit-image/en/205/generated/gallery/radial_histogram_equalization.html#sphx-glr-generated-gallery-radial-histogram-equalization-py

Reply:
If you pass in no clip_internal and pass in `norm=None` what does the map plot look like?

Reply:
perfect! That made the frames match. Will the user always have to do that, or is there a way to make that default somehow?

Reply:
A user would have to do that always unless we set the norm t to be that returned map in the function.